CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/QualityControlViewPane.java

```diff
@@ -412,8 +413,8 @@ public static String qualityStyle(final QualityCategory category, final float al
     private static void showPriorityDialog() {
         final ScrollPane rulesScrollPane = new ScrollPane();
         rulesScrollPane.setPrefHeight(240);
-        rulesScrollPane.setPrefWidth(750);
-        rulesScrollPane.setVbarPolicy(ScrollPane.ScrollBarPolicy.ALWAYS);
+        rulesScrollPane.setPrefWidth(SystemUtils.IS_OS_LINUX ? 820 : 700);
+        rulesScrollPane.setVbarPolicy(ScrollPane.ScrollBarPolicy.NEVER);
```

Comment:
Might be good to change to `AS_NEEDED`. While Core doesn't have many rules, if someone wants to expand upon and add their own rules, it may be once they have enough rules that they can't access those rules in the dialog (without expanding the dialog size)